lib/kernel/doc/src/code.xml

```diff
@@ -614,7 +617,11 @@ ok = code:finish_loading(Prepared),
       <desc>
         <p>Tries to load any modules not already loaded in the list
 	<c><anno>Modules</anno></c> in the same way as
-          <seemfa marker="#load_file/1">load_file/1</seemfa>.</p>
+          <seemfa marker="#load_file/1">load_file/1</seemfa>.
+          Opposite to <seemfa marker="#ensure_loaded/1">ensure_loaded/1</seemfa>,
+          concurrent calls to this function will attempt to load
+          modules concurrently. Furthermore, modules are loaded
+          even in <c>embedded</c> mode.</p>
```

Comment:
After discussion with @jhogberg we decided that we should not go into details about what happens in concurrent scenarios (at least not now). No other part of the code server documentation include those details:

```suggestion
          <seemfa marker="#load_file/1">load_file/1</seemfa>.
          Unlike <seemfa marker="#ensure_loaded/1">ensure_loaded/1</seemfa>,
          modules are loaded even in <c>embedded</c> mode.</p>
```

Reply:
Should I also remove mentions to concurrency scenarios from `ensure_loaded/1`?